src/main/java/uk/gov/ons/ctp/response/casesvc/client/CollectionExerciseSvcClient.java

```diff
@@ -101,10 +103,11 @@ public List<CollectionExerciseDTO> getCollectionExercises(final String surveyId)
    */
   public void createCollectionExercise(
       final UUID surveyId, final String exerciseRef, final String userDescription) {
-    log.with("survey_id", surveyId)
-        .with("exercise_ref", exerciseRef)
-        .with("user_description", userDescription)
-        .debug("Creating a collection exercise");
+    log.debug(
+        "Creating a collection exercise",
+        kv("survey_id", surveyId),
+        kv("exercise_ref", exerciseRef),
+        kv("user_description", userDescription));
```

Comment:
Slightly different pattern for including key-value pairs and the signature of the logging methods. These are now passed as parameters instead of being chained in the log object. You will see these changes throughout this PR.